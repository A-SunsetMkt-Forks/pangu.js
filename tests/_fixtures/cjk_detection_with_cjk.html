<!DOCTYPE html>
<html>
<head>
  <title>Page with CJK</title>
</head>
<body>
  <p id="test">新八的構造成分有95%是眼鏡、3%是水、2%是垃圾</p>
</body>
</html>