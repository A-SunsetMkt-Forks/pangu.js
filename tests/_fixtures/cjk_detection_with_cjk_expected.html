<p id="test">新八的構造成分有 95% 是眼鏡、3% 是水、2% 是垃圾</p>